doc/userguide/rules/dhcp-keywords.rst

```diff
@@ -6,6 +6,8 @@ dhcp.leasetime
 
 DHCP lease time (integer).
 
+dhcp.leasetime uses an :ref:`unsigned 64-bits integer <rules-integer-keywords>`.
```

Comment:
`64-bits` or `64-bit`?

Reply:
Without an s if I believe 

> 16-bit integers are those that are 16 bits (2 octets) wide.

Fixing that